Tools/baselineDef.cc

```diff
@@ -204,8 +296,8 @@ bool BaselineVessel::PredefineSpec()
   }
   else if(spec.compare("Zinv") == 0 || spec.compare("Zinv1b") == 0 || spec.compare("Zinv2b") == 0 || spec.compare("Zinv3b") == 0 || spec.compare("ZinvJEUUp") == 0 || spec.compare("ZinvJEUDn") == 0 || spec.compare("ZinvMEUUp") == 0 || spec.compare("ZinvMEUDn") == 0) 
   {
-    jetVecLabel = "jetsLVecLepCleaned";
-    CSVVecLabel = "recoJetsBtag_0_LepCleaned";
+    jetVecLabel = "prodJetsNoLep_jetsLVec";
```

Comment:
Should we use "bool BaselineVessel::UseLepCleanJets() " here instead

Reply:
Actually we should discuss whether this function is still needed? This is inherited from previous analysis, where each background config it here and I am not maintaining it anymore.

Reply:
I will talk to @caleb-james-smith to determine how this will be handled going forward.